polymer-redux.js

```diff
@@ -19,11 +19,12 @@
                     var splices = [];
                     var value, previous;
 
-                    // statePath a path or function
-                    if (typeof property.path == 'function') {
-                        value = property.path.call(element, state);
+                    // statePath, a path or function.
+                    var path = property.path;
+                    if (typeof path == 'function') {
+                        value = path.call(element, state);
                     } else {
-                        value = Polymer.Base.get(property.path, state);
+                        value = Polymer.Base.get(path, state);
```

Comment:
It would be better to stick with `Polymer.Base.get`, better tested and also can be used with arrays.